baseapp/baseapp.go

```diff
@@ -71,25 +70,28 @@ type BaseApp struct {
 	voteInfos    []abci.VoteInfo // absent validators from begin block
 
 	// consensus params
-	// TODO move this in the future to baseapp param store on main store.
+	// TODO: Move this in the future to baseapp param store on main store.
```

Comment:
Is there an issue open for this work?

Reply:
Man I have no idea -- this code is ancient, way before my time. I'll have to do a search or create an issue.